src/libraries/System.Private.CoreLib/src/System/BitConverter.cs

```diff
@@ -183,6 +183,25 @@ public static bool TryWriteBytes(Span<byte> destination, ulong value)
             return true;
         }
 
+        // Converts a half into an array of bytes with length
```

Comment:
Could you please add full Xml comments to the new methods, so that we can generate the official documentation from them later?

Reply:
Well, the existing methods don't have xml documents. I think they should be updated together.

Reply:
Updating or adding XML documentation on existing APIs is nice-to-have. We do require the XML documentation on any new APIs.